sanic/deprecated.py

```diff
@@ -0,0 +1,102 @@
+# NOTE(tomaszdrozdz): remove in version: 21.3
+# We replace from_envvar(), from_object(), from_pyfile() config object methods
+# with one simpler update_config() method.
+# We also replace "loading module from file code" in from_pyfile()
+# in a favour of load_module_from_file_location().
+# Please see pull request: 1903
+# and issue: 1895
+import types
+
+from warnings import warn
+
+from sanic.exceptions import PyFileError
+from sanic.helpers import import_string
+
+from typing import Any
+
+
+def from_envvar(self, variable_name: str) -> bool:
+    """Load a configuration from an environment variable pointing to
+    a configuration file.
+
+    :param variable_name: name of the environment variable
+    :return: bool. ``True`` if able to load config, ``False`` otherwise.
+    """
+
+    warn(
+        "Using `from_envvar` method is deprecated and will be removed in v21.3, use `app.update_config` method instead.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+
+    config_file = os_environ.get(variable_name)
```

Comment:
You should import `os` package and then use `os.environ.get()` the method because right now, we don't have it, and that's the reason why has been failing.

Reply:
Right - forgot about import os.environ.
But personally here I would prefer to import only environ from os - there is no need to import all os - it should be also a bit faster i suppose ?

Reply:
Not sure that it is faster, but I am fine with:
```
from os import environ
```

Reply:
OK.